demo/DemoArticles/ArticleRef.js

```diff
@@ -0,0 +1,25 @@
+/*
+#!/usr/bin/env node
+*/
+/* eslint-disable require-jsdoc */
```

Comment:
I recommend adding simple typing docs like `@return {!string}` and nothing more.